source/reference/service-limitations.txt

```diff
@@ -189,6 +190,26 @@ Reading:
    When you intend to have many concurrent writers, have each writer
    work on a separate document. 
 
+.. _mongodb-service-limitations-sync-default-message-size-limit:
+
+Default Message Size Limit
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Device Sync has a default message size limit of {+sync-default-message-size-limit+}
+compressed for any messages sent from the client application. The message size
```

Comment:
Extra "compressed" here?

```suggestion
for any messages sent from the client application. The message size
```

Reply:
I am not seeing an extra compressed here - is there something you're seeing that I'm missing?

Reply:
The HELP ticket that is linked from this Jira ticket refers to both compressed and uncompressed size. Uncompressed size of the specific transaction was 83 MB, compressed size was 23 MB. The Sync Protocol docs go into detail about compression and delta sync: https://www.mongodb.com/docs/atlas/app-services/sync/details/protocol/#changeset

I think the point is to note that the size limit is based on the size of the changeset after the compression has been applied. So changesets that are greater than 20 MB on the client may be fine as long as the compressed size falls within the limit.

Reply:
Gotcha. Maybe it makes sense to mention that the data is compressed automatically? Otherwise some users may think they need to manually compress the data.

Reply:
It's a fair point that this isn't explained here and it would be easy to misinterpret. I added some additional verbiage and linked out to the relevant docs section for more info about the compression we use. Thanks for the feedback!